sc2/client.py

```diff
@@ -1,29 +1,26 @@
-from s2clientprotocol import (
-    sc2api_pb2 as sc_pb,
-    common_pb2 as common_pb,
-    query_pb2 as query_pb,
-    debug_pb2 as debug_pb,
-    raw_pb2 as raw_pb,
-)
-
 import logging
-
+import pickle
+from typing import Any, Dict, Iterable, List, Optional, Set, Tuple, Union
+
+from s2clientprotocol import common_pb2 as common_pb
+from s2clientprotocol import debug_pb2 as debug_pb
+from s2clientprotocol import query_pb2 as query_pb
+from s2clientprotocol import raw_pb2 as raw_pb
+from s2clientprotocol import sc2api_pb2 as sc_pb
 from sc2.ids.ability_id import AbilityId
```

Comment:
@tweakimp could you also change these to
```
from .ids.ability_id import AbilityId
from .ids.unit_typeid import UnitTypeId
from .renderer import Renderer
```

It's breaking my relative import from `/lib/sc2`. I know it's not part of your PR but it needs to be cleaned up anyways to fit the rest of python-sc2's imports.

Reply:
you are right, its incosistent

Reply:
Fixed in https://github.com/Dentosal/python-sc2/pull/233/commits/1245c064388c45f49089a450c6503e6c14345f7c